src/App.tsx

```diff
@@ -1,20 +1,17 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
-
+import { Outlet } from 'react-router-dom';
```

Comment:
Use either single quotes or double quotes everywhere, keep the style of the code.